client/lib/route/page-notifier.js

```diff
@@ -1,23 +1,34 @@
 /*eslint no-multi-spaces: 0*/
 // node dependency
 var EventEmitter   = require( 'events' ).EventEmitter;
+var ipc = require( 'electron' ).ipcRenderer;
+var includes = require( 'lodash' ).includes;
 
 // internal dependency
-var page           = require( 'page' );
-var Route          = page.Route;
+var page		   = require( 'page' );
+var Route		  = page.Route;
```

Comment:
The spacing is odd here. There's a mix of tabs and spaces with the intent to align the right side of the statement. But this isn't something that should be aligned anyway. Should be single spaces. It'd be helpful to separate spacing changes to a separate commit so it's easier to parse out the relevant changes.